mongodb/mongodb-config.yaml

```
@@ -0,0 +1,59 @@
+apiVersion: mongodbcommunity.mongodb.com/v1
+kind: MongoDBCommunity
+metadata:
+  name: mongodb
+spec:
+  members: 3
+  security:
+    authentication:
+      modes:
+      - SCRAM
+  statefulSet:
+    spec:
+      selector: {}
```

```suggestion
      selector: {}
      template:
        spec:
          containers:
            - name: "mongodb-agent"
              resources:
                requests:
                  cpu: 100m
                  memory: 100M
                limits:
                  cpu: 500m
                  memory: 500M
            - name: "mongod"
              resources:
                requests:
                  cpu: 100m
                  memory: 500M
                limits:
                  cpu: 1000m
                  memory: 1000M  
```

These more-conservative-than-default limits will enable mongodb to run on a lower-resource cluster (eg. 3x m5a.large) alongside Ondat.